exaudfclient/base/exaudflib/impl/socket_high_level.cc

```diff
@@ -0,0 +1,497 @@
+#include "../exaudflib.h"
+#include "exaudflib/impl/socket_low_level.h"
+#include "exaudflib/zmqcontainer.pb.h"
+#include "exaudflib/impl/msg_conversion.h"
+#include "exaudflib/impl/global.h"
+#include <sys/resource.h>
+#include "debug_message.h"
+
+
+//static exascript_vmtype vm_type;
+static exascript_request request;
+static exascript_response response;
+static std::string output_buffer;
+
```

Comment:
is there a reason why these are not it the c++ namespace exaudflib, this would simplify a bit the usage of global variable

Reply:
Done